packages/ingest/src/singleLessonDryRun.ts

```diff
@@ -0,0 +1,113 @@
+import { CompletedLessonPlanSchemaWithoutLength } from "@oakai/aila/src/protocol/schema";
+import fs from "node:fs";
+import path from "node:path";
+import { zodResponseFormat } from "openai/helpers/zod.mjs";
+import { dirname } from "path";
+import { fileURLToPath } from "url";
+
+import { getCaptionsByFileName } from "./captions/getCaptionsByFileName";
+import { getCaptionsFileNameForLesson } from "./captions/getCaptionsFileNameForLesson";
+import { getSystemPrompt } from "./generate-lesson-plans/getSystemPrompt";
+import { getUserPrompt } from "./generate-lesson-plans/getUserPrompt";
+import { graphqlClient } from "./import-lessons/graphql/client";
+import {
+  query,
+  type QueryVariables,
+  type QueryWhere,
+} from "./import-lessons/graphql/query";
+import { openai } from "./openai-batches/openai";
+import { type RawLesson, RawLessonSchema } from "./zod-schema/zodSchema";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+void singleLessonDryRun();
```

Comment:
Should this be after the definition?

Reply:
definition is hoisted but can move to bottom if it reads better?